src/controllers/base/LessController.php

```diff
@@ -98,8 +98,16 @@ public function actionUpdate($id)
         $model = $this->findModel($id);
         if ($model->load($_POST) && $model->save()) {
             Yii::$app->session->addFlash('success', 'Record has been updated');
-            if (ArrayHelper::getValue($_POST, 'subaction') != 'apply') {
-                return $this->redirect(['view', 'id' => $model->id]);
+            if (ArrayHelper::getValue($_POST, 'subaction') === 'lint') {
+                $model->lintLess();
+            }
+            if (ArrayHelper::getValue($_POST, 'subaction') === 'fix') {
+                $model->lintLess(true);
+                $model->value = $model->fixedValue;
+                $model->save();
+            }
+            if (ArrayHelper::getValue($_POST, 'subaction') === 'save') {
+                return $this->redirect(Url::previous());
```

Comment:
This could lead to some unexpected behavior if Url:: previous() is null.

Try
```php
return $this-redirect(['update', 'id' => $model->id]);
```

Reply:
Yes, please do not use `Url::previous` & `Url::remember` here, they are a source of unexpected behaviors in this use-case

Reply:
it seems that the standard behaviour after save is to remain in the same page ('update/id').
Maybe i should redirect to 'index' when 'Save' is clicked

Reply:
it should go to view/{id}